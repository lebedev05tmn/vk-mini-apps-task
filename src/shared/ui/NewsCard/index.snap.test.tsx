import { NewsCard } from "./index";

const ReactTestRenderer = require("react-test-renderer");

describe("News Card render", () => {
  test("Should NewsCard render", () => {
    const tree = ReactTestRenderer.create(
      <NewsCard
        by={"author"}
        time={"12345"}
        onClick={() => console.log("hello world")}
        index={1}
        title={"title"}
        score={15}
      />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
