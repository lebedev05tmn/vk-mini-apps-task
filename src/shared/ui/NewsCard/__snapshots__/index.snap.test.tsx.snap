// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`News Card render Should NewsCard render 1`] = `
<div
  className="vkuiCard"
  onClick={[Function]}
>
  <span
    className="vkuiTypography vkuiTypography--normalize vkuiTitle--level-1"
  >
    <span>
      1
      .
    </span>
    title
  </span>
  <span
    className="vkuiTypography vkuiParagraph"
  >
    Рейтинг: 
    15
  </span>
  <span
    className="vkuiTypography vkuiParagraph"
  >
    Ник автора: 
    author
  </span>
  <span
    className="vkuiTypography vkuiParagraph"
  >
    Дата публикации: 
    12345
  </span>
</div>
`;
